apischeme_SSS.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Please set this repository up with `flake8` at the outset.

```
[efried@efried private-cluster-rhapi-apischeme-updater]$ flake8 .|wc -l
46
```

(Many of those are in generate_template.py -- see below.)

Reply:
I cleaned up most of this :) Thank you for the suggestion!

Reply:
Thanks. I hope to get flake8 enforcement integrated into all python-havin projects' builds. Needs some cultural adaptation though. See https://github.com/openshift/managed-cluster-validating-webhooks/pull/47